resources/views/plugins.blade.php

```diff
@@ -6,7 +6,7 @@
             @if($file['type'] == $type && $type == 'css')
                 <link rel="stylesheet" href="{{ $file['asset'] ? asset($file['location']) : $file['location'] }}">
             @elseif($file['type'] == $type && $type == 'js')
-                <script src="{{ $file['asset'] ? asset($file['location']) : $file['location'] }}"></script>
+                <script src="{{ $file['asset'] ? asset($file['location']) : $file['location'] }}" {{ (isset($file['defer']) && $file['defer']) ? 'defer' : '' }} ></script>
```

Comment:
I think you can change the two conditions by `{{ ! empty($file['defer']) ? 'defer' : '' }}`. Another solution would be:

```blade
<script src="{{ $file['asset'] ? asset($file['location']) : $file['location'] }}" @if(! empty($file['defer'])) defer @endif></script>
```

Reply:
OK. Done.